x-pack/plugins/cloud_security_posture/tsconfig.json

```diff
@@ -45,6 +45,7 @@
     "@kbn/es-types",
     "@kbn/core-elasticsearch-server",
     "@kbn/ecs",
+    "@kbn/core-saved-objects-api-server-mocks",
```

Comment:
@orouz @CohenIdo I am not sure that this config should be pushed - what do you think?

Reply:
Never mind it's not one of my changes, it came from `main`